GPU/Directx9/DrawEngineDX9.cpp

```diff
@@ -618,7 +622,7 @@ void DrawEngineDX9::DoFlush() {
 
 		ApplyDrawStateLate();
 
-		VSShader *vshader = shaderManager_->ApplyShader(false, false, lastVType_, decOptions_.expandAllWeightsToFloat, decOptions_.applySkinInDecode, pipelineState_);
+		VSShader *vshader = shaderManager_->ApplyShader(false, false, lastVType_, decOptions_.expandAllWeightsToFloat, true, pipelineState_);
```

Comment:
I didn't really need to force these to true, I just decided it's clearer since many of the paths have hardware transform also forced true/false.

-[Unknown]

Reply:
Fine with me.